### ISSUE TITLE
mounts broken in ambassador-telepresence-agent:1.13.14

### ISSUE THREAD

**Issue by nicks** [telepresenceio/telepresence#3221]
**Describe the bug**
Mounts no longer work in version 1.13.14 of the agent

If I flip to version 1.13.13, mounts work again.

**To Reproduce**

Run these commands:

```
$ telepresence intercept -n avatars api --port 5000 --mount=true -- bash
...
$ find "$TELEPRESENCE_ROOT"
/tmp/telfs-2668712529
$ kubectl get pods -n avatars -o yaml | grep "agent:1.13"
      image: docker.io/datawire/ambassador-telepresence-agent:1.13.14
      image: docker.io/datawire/ambassador-telepresence-agent:1.13.14
```

If I update the client configuration to use ambassador-telepresence-agent:1.13.13, the commands work again

```
$ telepresence intercept -n avatars api --port 5000 --mount=true -- bash
...
$ find "$TELEPRESENCE_ROOT" | grep kubernetes.io.serviceaccount
/tmp/telfs-3397515821/var/run/secrets/kubernetes.io/serviceaccount
/tmp/telfs-3397515821/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
/tmp/telfs-3397515821/var/run/secrets/kubernetes.io/serviceaccount/token
/tmp/telfs-3397515821/var/run/secrets/kubernetes.io/serviceaccount/namespace
...
$ kubectl get pods -n avatars -o yaml | grep "agent:1.13"
      image: docker.io/datawire/ambassador-telepresence-agent:1.13.13
      image: docker.io/datawire/ambassador-telepresence-agent:1.13.13
```

**Expected behavior**
I expected the mounts to work

**Versions (please complete the following information):**

```
telepresence version
An update of telepresence from version 2.13.1 to 2.13.3 is available. Please visit https://www.getambassador.io/docs/telepresence/latest/install/upgrade/ for more info.
Client         : v2.13.1
Root Daemon    : v2.13.1
User Daemon    : v2.13.1
Traffic Manager: v2.13.1
```

linux/amd64

tested on k3d and EKS

**Comment by thallgren** [comment]
Did you try updating the client/traffic-manager to v2.13.3? Not that it should be necessary (the agent should be backward compatible), but knowing this might help narrowing down the problem.

**Comment by nicks** [comment]
Ya, updating to 2.13.3 was the first thing I tried but didn't appear to fix it.

**Comment by thallgren** [comment]
This will be fixed in 1.13.16. A release candidate, 1.13.16-rc.0 is available now.
